Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Editor/CompositorWindow.cs

```diff
@@ -0,0 +1,346 @@
+                            Graphics.DrawTexture(quadrantRects[0], compositionManager.TextureManager.compositeTexture);
+                        }
+
+                        if (compositionManager.TextureManager.colorRGBTexture != null)
+                        {
+                            Graphics.DrawTexture(quadrantRects[1], compositionManager.TextureManager.colorRGBTexture, compositionManager.TextureManager.IgnoreAlphaMaterial);
+                        }
+
+                        if (compositionManager.TextureManager.renderTexture != null)
+                        {
+                            Graphics.DrawTexture(quadrantRects[2], compositionManager.TextureManager.renderTexture, compositionManager.TextureManager.IgnoreAlphaMaterial);
+                        }
+
+                        if (compositionManager.TextureManager.alphaTexture != null)
+                        {
+                            Graphics.DrawTexture(quadrantRects[3], compositionManager.TextureManager.alphaTexture, compositionManager.TextureManager.IgnoreAlphaMaterial);
+                        }
+                    }
+                    else
+                    {
+                        Graphics.DrawTexture(framesRect, compositionManager.TextureManager.compositeTexture);
+                    }
+                }
+            }
+        }
+
+        private void RecordingGUI()
+        {
+            recordingFoldout = EditorGUILayout.Foldout(recordingFoldout, "Recording");
+            if (recordingFoldout)
+            {
+                CompositionManager compositionManager = GetCompositionManager();
+
+                EditorGUILayout.BeginVertical("Box");
+                {
+                    RenderTitle("Recording", Color.green);
+
+                    GUI.enabled = compositionManager != null && compositionManager.TextureManager != null;
+                    if (compositionManager == null || !compositionManager.IsRecording())
+                    {
+                        if (GUILayout.Button("Start Recording"))
+                        {
+                            compositionManager.StartRecording();
+                        }
+                    }
+                    else
+                    {
+                        if (GUILayout.Button("Stop Recording"))
+                        {
+                            compositionManager.StopRecording();
+                        }
+                    }
+                }
+
+                if (GUILayout.Button("Take Picture"))
+                {
+                    compositionManager.TakePicture();
+                }
+
+                EditorGUILayout.Space();
+                GUI.enabled = true;
+
+                // Open Folder
+                if (GUILayout.Button("Open Folder"))
+                {
+                    Process.Start(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments), "HologramCapture"));
+                }
+
+                EditorGUILayout.EndVertical();
```

Comment:
>EditorGUILayout.EndVertical(); [](start = 16, length = 30)

Should this be just below the curly at 177? Or rather the code above should be above that curly.

Reply:
Good catch!